File: gaia-sdk-python/graphql/response/type/CreatedIntentImpulse.py

from graphql.response.type.Intent import Intent


class CreatedIntentImpulse:
    """
    Impulse which indicates the resulf of a create intent impulse
    """
    def id(self) -> Uuid:
        return self.id
    """
    the intent instance
    """
    def intent(self) -> Intent:
        return self.intent

File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/CreatedIntentImpulse.kt
package gaia.sdk.response.type

import gaia.sdk.Uuid
import gaia.sdk.ISO8601
import gaia.sdk.Struct
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

/**
* Impulse which indicates the result of a create intent impulse
*/
data class CreatedIntentImpulse @JsonCreator constructor(
    @JsonProperty("id") val id:Uuid? = null, 
    /**
    * the intent instance
    */
    @JsonProperty("data") val data:Intent? = null
)